stackrox/stackrox
# Replace type assertions with generic type parameters in tableColumnDescriptor for integrations

## Description

Follow suggestion by Dave in https://github.com/stackrox/stackrox/pull/2701#pullrequestreview-1073059908

## Checklist

## Testing Performed

### manual testing with feature flag

1. `export ROX_QUAY_ROBOT_ACCOUNTS=true` in ui
2. `yarn deploy-local` in ui
3. `yarn start` in ui

    * Visit /main/integrations/imageIntegrations/clairify: See table does have a **Type** column

    * Visit /main/integrations/imageIntegrations/quay: See table does have a **Type** column
